app/assets/javascripts/event_lister.js

```diff
@@ -0,0 +1,19 @@
+$(document).on("ready page:load", function() {
+
+  var createEvent = function () {
+    var userId = $(this).attr("data-user");
+    var eventId = $(this).attr("data-event");
+    var request = $.ajax({
+      url: "/api/v1/user_events.json",
+      method: "POST",
+      data: { user_event: { user_id: userId, event_id: eventId } }
```

Comment:
Identifier 'user_event' is not in camel case.<br>Identifier 'user_id' is not in camel case.<br>Identifier 'event_id' is not in camel case.